crates/embucket-lambda/src/main.rs

```diff
@@ -140,9 +144,9 @@ impl LambdaApp {
             );
         }
 
-        if let Err(err) = ensure_session_header(&mut parts.headers, &self.state).await {
-            return Ok(snowflake_error_response(&err));
-        }
+        // if let Err(err) = ensure_session_header(&mut parts.headers, &self.state).await {
```

Comment:
Just drop session header, but leave ensure_session - if no session found, we still need to create one.